### ISSUE TITLE
Possible conflict with weak aura weaon enchant detection

### ISSUE THREAD

**Issue by linaori** [SLOKnightfall/BetterWardrobe#127]
The issue can be read here: https://github.com/WeakAuras/WeakAuras2/issues/2743
TL;DR: in certain scenarios the Flametongue weapon enchant can't be detected properly

The issue can be seen here, the cooldown remaining is missing. I print all the the tooltip lines in the weak aura trigger: https://github.com/WeakAuras/WeakAuras2/blob/12499cdb9179281fce2bd95ecaf67fb5e1b6d975/WeakAuras/GenericTrigger.lua#L3089-L3090
![image](https://user-images.githubusercontent.com/1754678/99840947-b7cc9600-2b6d-11eb-8d8a-74a61f7a78c1.png)
(left=reload, right=login)

For some reason when I enable the Tooltip addon, when logging in this value is missing the timer. This is only the case when the tooltips addon is enabled. I'm not sure where or why this is going wrong. This also randomly happens on other moments (as described in the link).

The following "fixes" the issue:
```diff
local frame = CreateFrame("FRAME");
frame:RegisterEvent("PLAYER_ENTERING_WORLD");
local function eventHandler(self, event, ...)
+ --LoadAddOn("Blizzard_Collections")
- LoadAddOn("Blizzard_Collections")
end
frame:SetScript("OnEvent", eventHandler);
```

**Comment by SLOKnightfall** [comment]
Removing that line breaks this addon.  When it was originally written it was written to load when the Collection Journal was opened/loaded. Now that it has expanded to do other things outside the Collection Journal it still needs to have the Collection Journal be loaded first work and the "Tooltip" addon was created to load the Collection Journal during start up so this addon can also load.  

This addon adds certain information to the bottom of the tooltip in regards to if a appearance is learned or if it is part of a set.  If there was an incompatibility you most likely wouldn't be seeing your weak aura tooltip.  As the weak aura tooltip is being shown but with the timer missing, that makes me think that there is an issue where the weak aura script is not correctly reading the timer at login.  I'll look into the tooltip code to see if I can find anything but cant thing of why it cause the issue you described.   The game has been quite laggy lately and I have seen tooltips appear to not have all the data loaded before the tooltip was shown resulting in only partial data being shown.

**Comment by linaori** [comment]
> This addon adds certain information to the bottom of the tooltip in regards to if a appearance is learned or if it is part of a set.

Yeah, I expected this issue to be gone turning off all the tooltip options, but that didn't solve it.
![image](https://user-images.githubusercontent.com/1754678/99877998-70441980-2c02-11eb-814d-689d88ea6fff.png)

> As the weak aura tooltip is being shown but with the timer missing, that makes me think that there is an issue where the weak aura script is not correctly reading the timer at login

The screenshots I've posted are from the weak aura and shows the tooltip at that point as it, fetched from the blizzard API I believe. The match of the weak aura requires the timer to be available in order to detect it as an enchant. This happens before I'm done loading, so there's not actually a tooltip being displayed yet. I'm not sure what could be causing the trim.

>  The game has been quite laggy lately and I have seen tooltips appear to not have all the data loaded before the tooltip was shown resulting in only partial data being shown.

One thing that crossed my mind was that this could indeed be related to the amount of addons. However, after turning off all addons except WA and better wardrobe + options, it still happened.

>  I'll look into the tooltip code to see if I can find anything but cant thing of why it cause the issue you described

Thanks! I've taken a peek myself, I just don't understand enough of how it works and where it replaces the tooltip part to see what could be wrong 😅
